doc/OPTIONS

```diff
@@ -394,37 +394,50 @@ their number could potentially be decreasing as some hashes get cracked.
 To have the cracked hashes (and possibly salts) removed from all
 processes, you may interrupt and restore the session once in a while.
 
+
 --format=NAME[,NAME...]		force hash type NAME
```

Comment:
As the section for --format is so large, I split it into paragraphs. For separating *that* from the rest, I added empty lines (so they are now double empty lines) above and below the whole section.

Reply:
Perhaps we should have empty lines between descriptions of _every_ option, then - for consistency.

Reply:
Yes, and actually we should review the whole file in case more sections should be split the same way. Also, the used line width varies a little, between below 70 to at least a full 80, which is not so good. Oh, and we're not consistently using double space after period.

That'd be a separate task though.